src/ModelManager.ts

```diff
@@ -552,6 +566,36 @@ export class ModelManager {
 
         return Promise.resolve();
     }
+
+    /**
+     * Fetches parent page information of the given component path
+     * Returns object containing
+     * 1. Parent page path
+     * 2. Parent page data if already available in the store
+     * @return {object}
+     * @private
+     */
+    private _fetchParentPage(path: string) {
```

Comment:
that's not how we do in typescript :)
- please declare return type (without comments if properties are self explaining), or maybe it's even declared somewhere
- no need to add `@private`

for example:
```
type PageData = {
    pageData: string;
    pagePath: string;
};

...

    /**
     * Returns model for specified path.
     */
    private _fetchParentPage(path: string): PageData {
```

Reply:
if `pageData` is not a `string`, please declare dedicated type :)